mintlify/mint
# Bump cli to 3.0.14

## Summary
title & update contributing.md
This version of the CLI bumps the previewing package back up to 3.1.4

## Test Plan
- `npm install -g mintlify`
- `mintlify` should print help message
- `mintlify -v` should be 3.0.14
- `npm ls -g` mintlify should be v3.0.14
- navigate to docs instance & sanity check `mintlify dev`
- try to follow the steps in contributing.md
